playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -182,16 +185,134 @@ EDXAPP_PDF_RECEIPT_TAX_ID_LABEL: "fake Tax ID"
 EDXAPP_PDF_RECEIPT_COBRAND_LOGO_PATH: ""
 EDXAPP_PDF_RECEIPT_LOGO_PATH: ""
 
+ACCOUNT_NAME: 'Stanford Online'
+ANALYTICS_DASHBOARD_NAME: 'Stanford Online Insights'
```

Comment:
rename this to `EDXAPP_ANALYTICS_DASHBOARD_NAME` and map it to `generic_env_config`